### DISCUSSION TITLE
七次落泪 | /cn/2022/03/seven-tears/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1445] [Comments]
https://yihui.org/cn/2022/03/seven-tears/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
谢谢！那段时间有件事让我非常焦虑，就是我在美国的签证身份已经过期近八个月，我只差大概三个星期就快要成黑户、丢工作甚至丢一切了。还有些私事也让我很头疼。

> ——原帖发布于 2022-07-23 05:32:06

**Reply by yihui** [reply]
很多事情对比起来都不叫事，这便是我写《丹佛夜影》的主要动机之一。我们总觉得自己惨到不能再惨，但听别人的故事，就会觉得别人惨多了。

> ——原帖发布于 2022-07-23 15:22:47

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
26 看 HTML 注释，我只提过一句。22 是私事，就不谈了。

> ——原帖发布于 2022-08-02 20:04:33

**Reply by coderkami1** [reply]
注释貌似没了

[a comment by an automated account is not shown]

**Comment by coderkami1** [comment]
我最近两次落泪都是在梦里，很神奇，之前都没在梦里有这么强烈到哭的悲伤情绪。
